src/modules/vendor/components/companies/ProfileInfo.vue

```diff
@@ -69,6 +232,99 @@ export default {
         this.tmpInput = null;
       }
     },
+    async getUsers () {
+      try {
+        this.users = await Users.list({ role: [CLIENT_ADMIN, COACH], company: this.company._id });
+      } catch (e) {
+        console.error(e);
+        this.users = [];
+      }
+    },
+    async createUser () {
+      try {
+        await Users.create(pickBy(this.newUser));
+        this.userCreationModal = false;
+        await this.getUsers();
+        NotifyPositive('Utilisateur enregistré');
+      } catch (e) {
+        console.error(e);
+        NotifyNegative('Erreur lors de la création de l\'utilisateur');
+      }
+    },
+    resetUserCreationForm () {
+      this.$v.newUser.$reset();
+      this.newUSer = Object.assign({}, clear(this.newUser));
+    },
+    emailError (validationObj) {
+      if (!validationObj.local.email.required) {
+        return REQUIRED_LABEL;
+      } else if (!validationObj.local.email.email) {
+        return 'Email non valide';
+      }
+      return '';
+    },
+    phoneNbrError (validationObj) {
+      if (!this.$_.get(validationObj, 'contact.phone.frPhoneNumber', null)) {
+        return 'Numéro de téléphone non valide';
+      }
+      return '';
+    },
+    async getRoles () {
+      try {
+        this.roles = await Roles.list({ name: [CLIENT_ADMIN, COACH] });
+      } catch (e) {
+        console.error(e);
+        this.roles = [];
+      }
+    },
+    async openUserCreationModal () {
+      await this.getRoles();
+      const coachRole = this.roles.find(role => role.name === COACH);
+      if (coachRole) this.newUser.role = coachRole._id;
+    },
+    async openUserEditionModal (user) {
+      await this.getRoles();
+      this.selectedUser = {
+        ...this.selectedUser,
+        ...pick(cloneDeep(user), ['_id', 'identity', 'local', 'contact', 'role']),
+      };
+      this.selectedUser.role = this.selectedUser.role.client._id;
```

Comment:
tu peux le mettre directement dans la definition de l'objet this.selectedUser non ?

Reply:
bien vu !